core/model/modx/sources/modfilemediasource.class.php

```diff
@@ -222,6 +222,7 @@ public function getContainerList($path) {
 
                         $modAuth = $this->xpdo->user->getUserToken($this->xpdo->context->get('key'));
 
+                        $preview = 1;
```

Comment:
Any particular reason `$preview` here is an integer and not a boolean? It seems to be used as a boolean further along the code..

Reply:
No. Changed.